op-chain-ops/genesis/db_migration.go

```diff
@@ -192,7 +192,7 @@ func MigrateDB(ldb ethdb.Database, config *DeployConfig, l1Block *types.Block, m
 	}
 
 	// Finally we migrate the balances held inside the LegacyERC20ETH contract into the state trie.
-	// Note that we do NOT delete the balances from the LegacyERC20ETH contract.
+	// We also delete the balances from the LegacyERC20ETH contract.
```

Comment:
Reference for this comment change: 

https://github.com/ethereum-optimism/optimism/blob/a6a14b706d76c91f5cfe0188a2dcb014bc86a936/op-chain-ops/ether/migrate.go#L64-L66

Reply:
The contract will return you ETH balance when you call `balanceOf` tho, that was a modification we made

Reply:
right but that's purely in the logic (bytecode) not storage afaiui.